doc/classes/PopupMenu.xml

```diff
@@ -133,8 +133,25 @@
 			<param index="4" name="accel" type="int" enum="Key" default="0" />
 			<description>
 				Adds a new multistate item with text [param label].
-				Contrarily to normal binary items, multistate items can have more than two states, as defined by [param max_states]. Each press or activate of the item will increase the state by one. The default value is defined by [param default_state].
+				Contrarily to normal binary items, multistate items can have more than two states, as defined by [param max_states]. The default value is defined by [param default_state].
 				An [param id] can optionally be provided, as well as an accelerator ([param accel]). If no [param id] is provided, one will be created from the index. If no [param accel] is provided, then the default value of 0 (corresponding to [constant @GlobalScope.KEY_NONE]) will be assigned to the item (which means it won't have any accelerator). See [method get_item_accelerator] for more info on accelerators.
+				[b]Note:[/b] Multistate items don't update their state automatically and must be done manually. See [method toggle_item_multistate], [method set_item_multistate] and [method get_item_state] for more info on how to control it.
+				Example usage:
+				[codeblock]
+				func _ready():
+				    add_multistate_item("Item", 3, 0)
+
+				    index_pressed.connect(func(index: int):
+				        toggle_item_multistate(index)
+				        match get_item_state(index):
+				            0:
+				                print("First state")
+				            1:
+				                print("Second state")
+				            2:
+				                print("Third state")
+				    )
```

Comment:
```suggestion
				            toggle_item_multistate(index)
				            match get_item_state(index):
				                0:
				                    print("First state")
				                1:
				                    print("Second state")
				                2:
				                    print("Third state")
				        )
```
Continuation blocks should have two indentations

Reply:
Fixed! thanks!